CMakeLists.txt

```diff
@@ -54,7 +59,7 @@ idf_component_register(SRCS ${srcs}
                        INCLUDE_DIRS ${includes_public}
                        PRIV_INCLUDE_DIRS ${includes_private}
                        PRIV_REQUIRES esp_netif # required by rndis_reports.c: #include "netif/ethernet.h"
-                       REQUIRED_IDF_TARGETS esp32s2 esp32s3
+                       REQUIRED_IDF_TARGETS ${supported_mcu}
```

Comment:
Let's remove this field and maintain the supported targets in `idf_component.yml` only